### DISCUSSION TITLE
Potential synergies between PDF4QT and Inkscape

### DISCUSSION THREAD

**Discussion by GerryT11** [JakubMelka/PDF4QT#187] [Ideas]
In a recent discussion about open source PDF editors on Mastodon, the Inkscape core developer of Martin Owens joined the discussion and expressed interest in PDF4QT and how it parses PDFs (the content/objects). Although PDF4QT is a PDF editor and Inkscape is a vector drawing program (converting/importing/exporting PDFs), there might be potential synergies. 

Please see Mark's post here (and the discussion above and below): https://mastodon.social/deck/@doctormo@floss.social/112594279155369158

**Comment by JakubMelka** [comment]
Hello, great. I think, when editor plugin is finished, it will be easy to convert PDF to SVG or SVG to PDF using my software.
